pkg/features/list.go

```diff
@@ -128,6 +128,15 @@ var (
 
 	// Intgrate with LLM for risk recommendations
 	LLMRiskRecommendation = registerFeature("Intgrate with LLM for risk recommendations", "ROX_LLM_RISK_RECOMMENDATION")
+
+	// AutolockAllProcessBaselines automatically locks all process baselines after the observation period
+	AutolockAllProcessBaselines = registerFeature("Automatically locks all process baselines after the observation period", "ROX_AUTOLOCK_ALL_PROCESS_BASELINES")
+
+	// Adds built-in policy to ensure that Red Hat images are signed by Red Hat Release Key
+	RedHatImagesSignedPolicy = registerFeature("Adds built-in policy to ensure that Red Hat images are signed by the Red Hat release key", "ROX_RED_HAT_IMAGES_SIGNED_POLICY", unchangeableInProd)
+
+	// Locks process baselines when their deployments leave the observation period
+	AutoLockProcessBaselines = registerFeature("Locks process baselines when their deployments leave the observation period", "ROX_AUTOLOCK_PROCESS_BASELINES")
```

Comment:
Not sure what is going on here.  I'm guessing one of the Autolocks needs to disappear and a rebase will fix the image one showing as a diff.

Reply:
This was a result of messing up a rebase. I have removed the extra feature flags.